app/Platform/Services/TicketListService.php

```diff
@@ -42,10 +44,17 @@ public function getFilterOptions(Request $request): array
             'achievement' => $validatedData['filter']['achievement'] ?? 'all',
             'mode' => $validatedData['filter']['mode'] ?? 'all',
             'developerType' => $validatedData['filter']['developerType'] ?? 'all',
+            'developer' => $validatedData['filter']['developer'] ?? 'all',
+            'reporter' => $validatedData['filter']['reporter'] ?? 'all',
         ];
     }
 
-    public function getSelectFilters(bool $showStatus = true, bool $showDevType = true, bool $showAchievementType = true): array
+    public function getSelectFilters(
+        bool $showStatus = true,
+        bool $showAchievementType = true,
+        bool $showDevType = true,
+        bool $showDeveloper = false,
+        bool $showReporter = false): array
     {
```

Comment:
nit: stylistically, don't we generally do
```php
): array {
```

Reply:
Yep. I missed this when I changed it to multi-line and `composer lint` apparently doesn't care.

Reply:
No worries. I've also found how it handles whitespace can sometimes be unexpected or frustrating.